src/index.js

```diff
@@ -670,7 +671,18 @@ const members = {
       return this.getAttribute('name');
     },
     set(name) {
-      return this.setAttribute('name', name);
+      const ret = this.__setAttribute('name', name);
```

Comment:
Have you checked to see if this will recurse because you're setting the prop from the `setAttribute()` override?